src/parser/core/modules/GlobalCooldown.js

```diff
@@ -7,15 +7,25 @@ import {getAction} from 'data/ACTIONS'
 import Module from 'parser/core/Module'
 import {Group, Item} from './Timeline'
 
-const MIN_GCD = 1500
-const MAX_GCD = 2500
+const MIN_GCD_IN_MILLIS = 1500
+const MAX_GCD_IN_MILLIS = 2500
+const BASE_GCD_IN_SECONDS = 2.5
```

Comment:
I'd almost prefer all this be stored in ms, without the var suffixes - `1000` is a whitelisted magic if nessecary.

Not gonna  kick a stink about it, the names just feel dirty.

Reply:
That's fair. Even if they're all in ms, I'd prefer the suffix, but it wasn't my file to begin with. I'll make these changes.